pkg/querier/tenantfederation/merge_queryable_test.go

```diff
@@ -84,6 +117,33 @@ func metricMatches(m model.Metric, selector labels.Selector) bool {
 	return selector.Matches(labels.FromStrings(labelStrings...))
 }
 
+type mockSeriesSet struct {
+	upstream storage.SeriesSet
+	warnings storage.Warnings
+	queryErr error
+}
+
+func (m *mockSeriesSet) Next() bool {
+	return m.upstream.Next()
+}
+
+// At returns full series. Returned series should be iteratable even after Next is called.
```

Comment:
```suggestion
// At returns full series. Returned series should be iterable even after Next is called.
```

Reply:
This actually seems to be coming from upstream: https://github.com/prometheus/prometheus/pull/8713